rocky/rocky/settings.py

```diff
@@ -283,7 +287,7 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
-SESSION_EXPIRE_SECONDS = 60 * 60 * 2  # 2 hours
+SESSION_EXPIRE_SECONDS = 60 * 60 * 2  # Why 2 hours?
```

Comment:
If I recall correctly there's been some input from the community about users getting kicked out of the application after some inactivity. Maybe 2 hours is too short?

Reply:
this is something we'd need/want to check with our security testers. @sijmen-ruwhof @ITNerdbox

Reply:
Before establishing an appropriate session timer, it is crucial to ascertain the implementation details. Specifically, we need to determine whether the time-out operates on a fixed duration irrespective of user activity or if it triggers after a period of user inactivity.

Additionally, it is important to consider the specific environment in question, such as distinguishing between production and staging environments. Different time-out settings may be applicable to each environment based on their respective requirements.

Furthermore, other factors can influence the session time-out, including company policies that may align with the principles of confidentiality, integrity, and availability (CIA) or business impact analysis (BIA) ratings. These dependencies should be taken into account to ensure the session time-out aligns with the overall security objectives and business needs.

<hr>

**Scenario**

Considering the scenario as a production environment within an organization where scanning and storing vulnerabilities and known weaknesses of internal systems or applications takes place, I would recommend the session **_idle_** time-out to be set at 15 minutes.

From a malicious perspective, gaining access to a system like this would be a primary objective during the execution phase of an attack.

<hr>

Since there are many dependencies on what the session time-out should be, it may be better to process this information during the installation process.

Reply:
It would be great if this would be configarable...

Reply:
The `SESSION_EXPIRE_SECONDS` defines the fixed duration, not the period after user activity. I think 2 hours has a good tradeoff between security and UX as default. I agree that this should be configurable using an env variable.